### PR TITLE
fix(user-defaults): setting user defaults to not wipe db

### PR DESCRIPTION
**Pull request by bassrock**
## Summary
Ensure that the tests do not try and wipe the core data db after it has been seeded already. Introduced in https://github.com/Pocket/pocket-ios/pull/414

* Also fixed some swiftlint issues..

### CODE REVIEW

File: Tests iOS/Support/LaunchArguments.swift
Comment (bassrock) [inline Tests iOS/Support/LaunchArguments.swift:59]: https://sarunw.com/posts/how-to-set-userdefaults-value-with-launch-arguments/

Comment (cyndichin) [reply Tests iOS/Support/LaunchArguments.swift:59]: thanks for linking resources in your PR 🔥 helpful to know the context :)

**Review by cyndichin — APPROVED** [review]
do we have a ticket to revert the clear core data code before we publish to MVP? changes look good, approved and will let bitrise tell us if these tests pass.

**Comment by bassrock** [comment]
@cyndichin Yea I'll make a ticket to remove it in a month.

**Comment by bassrock** [comment]
https://getpocket.atlassian.net/browse/IN-1173

**Pull request merged**
